src/test/resources/no/nav/familie/ef/sak/beregning_barnetilsyn_en_mnd.feature

```diff
@@ -0,0 +1,81 @@
+# language: no
+# encoding: UTF-8
+
+Egenskap: Beregning av barnetilsyn (en periode)
+
+  Scenario: Varierende utgifter og inntekter, satsperiode og antall barn for 2022
+
+    Gitt følgende data
```

Comment:
Blir det enklere å lese denne hvis man splitter opp den på lik måte som den andre cucumbertesten, som har delt opp de ulike periodeutgiftene?

Reply:
Vi hadde nesten lyst til å merge resultat inn i samme tabell så man slapp "rad" og kunne lese alt man trengte i en rad. 

Når vi delte kode med funksjonelle måtte vi også merge disse for at det skulle bli lettere å lese, men ...

Reply:
Det skjønte jeg ikke
Hva mener du er forskjellen mellom denne måte og den andre? For meg virker det som att vi gjør det samme på 2 ulike måter, og får dobbelt opp av kode som skal vedlikeholdes

Og då hver rad kun har verdi en av disse kolonnene, så blir det som en csv-test, hvilket vi prøvde å gå bort fra når vi tok i bruk cucumber for å få tydligere testdata og tester

Reply:
Vi tenkte vi skulle gjøre som OC foreslår under: 
> Nå som det ble så mange eksempler, ville det nok vært veldig fint med scenariomal, slik at man kan få både testvariabler og expected value på samme rad

Hver rad = en test => resultat på samme rad. 
Når vi snakket med funksjonelle var det slik det var lettest å presentere testen og slik vi fikk tilbake "fasit" fra kari i et excel-ark.   

Men kanskje også dele opp testen, soretere radene bedre så de gir mening. 

Tenker det blir en ny pr - forbedring av cucumber